src/flatc.cpp

```diff
@@ -97,6 +97,7 @@ std::string FlatCompiler::GetUsageString(const char *program_name) const {
     "                     Default value is \"T\"\n"
     "  --no-js-exports    Removes Node.js style export lines in JS.\n"
     "  --goog-js-export   Uses goog.exports* for closure compiler exporting in JS.\n"
+    "  --es6-js-export    Uses ECMAScript 6 export style lines in JS.\n"
```

Comment:
Can you also add this to Compiler.md ?

Reply:
Done! I didn't see that file previously. Thanks for pointing it out to me.